src/backend/controllers/resources-controller.js

```diff
@@ -53,9 +54,18 @@ class ResourcesController extends BaseController {
         this.data.record,
       ))
     }
+    this.storePayloadData(payload)
     return this.render('pages/edit', this.data)
   }
 
+  storePayloadData(payloadData) {
+    Object.keys(payloadData).forEach(key => {
+      if(this.data.record.params[key]) {
```

Comment:
accessing inner record members are not the responsibility of controllers. This should go to #update and #save methods within the BaseRecord